pwa/src/pages/users/index.tsx

```diff
@@ -1,21 +1,19 @@
 import * as React from "react"
-import Layout from "../../components/common/layout";
 import UsersTable from "../../components/users/usersTable";
 
 const IndexPage = () => {
   return (
-    <Layout title={"Users"} subtext={"An overview of your users"}>
-      <main>
-        <div className="row">
-          <div className="col-12">
-            <div className="page-top-item">
-              <UsersTable/>
-            </div>
+    <main>
+      <div className="row">
+        <div className="col-12">
+          <div className="page-top-item">
+            <UsersTable/>
```

Comment:
This doesn't seem to be working for me. Maybe, together with the other comment about users, it might be worth to add this to the JIRA issue as well.